examples/uikit-vite-ts/src/main.tsx

```diff
@@ -1,6 +1,8 @@
 import React from "react";
+
```

Comment:
I don't understand where in the config it states that `"react"` needs to be in its own group - but ok 🤔

Reply:
[From what I've read](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md#distinctgroup-boolean.) this happens by default when setting the `position` for `"react"` inside `pathGroups`  and `newlines-between` is set to `always`. However, we can disable this by using [`distinctGroup`](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md#distinctgroup-boolean). I set `distinctGroup` to `false` in order for `"react"` to not have its own group. 
Let me know what you think.